cldf/concepticon.csv

```diff
@@ -3960,3 +3960,6 @@ ID,Name,Description,Semantic_Field,Ontological_Category,Replacement_ID
 3973,DEPRESSED,"A mental state of despondency, unhappinness, or hopelessness.",Emotions and values,Property,
```

Comment:
It seems that some quotation marks were introduced in the descriptions of DEPRESSED, UNHAPPY, NERVOUS, COMPANION. But I didn't see them when we added the new concepts.

Reply:
@AnnikaTjuka, I think these are there, since we have proper CSV now, with a comma. So the quotation mark `"` just marks that this is a cell content, and commas inside that cell should not count as cell delimiters. The CSV writer in CSVW does this automatically, following the requirements of "typical" CSV format.

Reply:
Ah ok!